packages/perseus/src/widgets/radio/radio-component.new.tsx

```diff
@@ -164,78 +134,50 @@ class Radio extends React.Component<Props> implements Widget {
             },
         );
 
-        // alwaysUpdate={true} so that passage-refs findWidgets
-        // get called when the outer passage updates the renderer
-        // TODO(aria): This is really hacky
+        // This has been called out as a hack in the past.
         // We pass in a key here so that we avoid a semi-spurious
         // react warning when the ChoiceNoneAbove renders a
         // different renderer in the same place. Note this destroys
         // state, but since all we're doing is outputting
-        // "None of the above", that is okay.
-        // TODO(mdr): Widgets inside this Renderer are not discoverable through
-        //     the parent Renderer's `findWidgets` function.
+        // "None of the above", that is okay. Widgets inside this Renderer
+        // are not discoverable through the parent Renderer's `findWidgets` function.
+        // alwaysUpdate={true} so that passage-refs findWidgets
+        // get called when the outer passage updates the renderer
         return (
```

Comment:
I cleaned up these comments as I don't think the TODOs are linked to anyone anymore.